src/api/app/models/flag.rb

```diff
@@ -66,7 +66,7 @@ def default_status
       arch_flag = main_object.project.flags.where("flag = ? AND repo IS NULL AND architecture_id = ?", flag, architecture_id).first unless arch_flag
       all_flag = main_object.project.flags.where("flag = ? AND repo IS NULL AND architecture_id IS NULL", flag).first unless all_flag
     end
-    
+
```

Comment:
Looks like you introduced those whitespaces in 6ee6edf4fe03b27c7f2942 . Please squash this commit into the one that introduced this change